.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.188.0/containers/cpp/.devcontainer/base.Dockerfile
+
+ARG VARIANT="buster"
```

Comment:
Shouldn't this be the same as `VARIANT` in the json file?

Reply:
This is autogenerated by VSCode, the default C++ ubuntu 20.04 version is called "buster" on microsoft's upstream container hosting

Reply:
Weird. "buster" is a Debian release. I imagine this means things are still mostly compatible between them... Should be fine.